runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -2350,6 +2350,4 @@ ZZ                                     unprototyped call
   IHAPSA
   CVT DSECT=YES
 
-    END
-})dnl
-
+    END})
```

Comment:
I think this was clearer as it was; we just need to remove line 2355.
A similar comment applies to the next two files.

Reply:
Let me try that again, I could have sworn I tried something similar and it didn't work, but its possible I did it wrongly.